autoload/SpaceVim/mapping/space.vim

```diff
@@ -197,7 +197,8 @@ function! SpaceVim#mapping#space#init() abort
         \ ]
         \ , 1)
   let s:lnum = expand('<slnum>') + s:funcbeginline
-  call SpaceVim#mapping#space#def('nnoremap', ['w', 'M'], 'ChooseWinSwap',
+  call SpaceVim#mapping#space#def('nnoremap', ['w', 'M'], 
+        \ "execute {-> winnr('$')<=2 ? 'wincmd x' : 'ChooseWinSwap'}()",
         \ ['swap window',
```

Comment:
lamada is not supported in old vim.

Reply:
do you prefer if I changed it to have this mod support all versions, or make the mod only available to versions of vim/nvim that have lambda support?

Reply:
in old vim, we can use `exe string`.

Reply:
hey bud, I changed the mapped command to use `eval()` instead of the lambda now